db/seeds/development/user.seeds.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+[
+  {
+    email: 'test@example.com',
+    password: 'password'
+  }
+].each do |user|
+  User.find_or_create_by(email: user[:email]).update(user)
```

Comment:
Let's raise an error if we can't create the user.
```suggestion
  User.find_or_create_by!(email: user[:email]).update(user)
```